config/sync/core.entity_form_display.node.voter_guide.default.yml

```diff
@@ -6,7 +6,7 @@ dependencies:
     - field.field.node.voter_guide.body
     - field.field.node.voter_guide.field_basics_block
     - field.field.node.voter_guide.field_icon
-    - field.field.node.voter_guide.field_media
+    - field.field.node.voter_guide.field_icon_hover
```

Comment:
This is a regression we need the `field_media` for the voter guide hero image.

Reply:
We already have a field for icon. What is this new icon_hover field? We can control the color of the icon using css.